### ISSUE TITLE
support new consecrated ground (100% increased crit chance to enemies)

### ISSUE THREAD

**Issue by zmike** [Openarl/PathOfBuilding#1442]
I'd like to try and help out with this one if it isn't already being worked on?

So far I've done this: https://gist.github.com/zmike/6b1374b0245824bca3f4cf961d0c5538

* create new mod (SelfIncCritChance) for "increased chance of taking crit" since this doesn't exist
* add 100 to SelfIncCritChance when OnConsecratedGround is set, since we can assume that anyone toggling this wants both effects
* attempt to implement the calculation in the crit section of dps calcs

But the calculation is probably not right since that doubles the effective crit chance, which is I assume not what would be intended here...

**Comment by LucasAzais** [comment]
I agree that doubling the crit chance would be crazy, although per your understanding the calculation seems correct. However I don't think it should be read as "increased chance of taking crit".

The patch note says: "Consecrated Ground now affects ... enemies (causing hits against them to have 100% increased critical strike chance). 
The way I understand it, it increases the crit chance of the hit itself by 100% and therefore doesn't work like "take increased damage" but rather like a simple +100% Increased Critical Strike Chance.

**Comment by zmike** [comment]
Oh I see, I misinterpreted it then. Thanks for the suggestion!
